addons/binding/org.openhab.binding.sensebox/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,195 @@
+            <channel id="imageUrl" typeId="imageUrl"/>
+            <channel id="mapUrl" typeId="mapUrl"/>
+        </channels>
+    </channel-group-type>
+
+    <channel-group-type id="measurements">
+        <label>Measurements</label>
+        <description>Measurements as fetched from the API</description>
+        <channels>
+            <channel id="uvIntensity" typeId="uvIntensity"/>
+            <channel id="luminance" typeId="luminance"/>
+            <channel id="pressure" typeId="pressure"/>
+            <channel id="humidity" typeId="humidity"/>
+            <channel id="temperature" typeId="temperature"/>
+        </channels>
+    </channel-group-type>
+
+    <channel-group-type id="lastReported">
+        <label>Last Reported</label>
+        <description>Timestamps when a measurement was last reported</description>
+        <channels>
+            <channel id="uvIntensityLastReported" typeId="uvIntensityLastReported"/>
+            <channel id="luminanceLastReported" typeId="luminanceLastReported"/>
+            <channel id="pressureLastReported" typeId="pressureLastReported"/>
+            <channel id="humidityLastReported" typeId="humidityLastReported"/>
+            <channel id="temperatureLastReported" typeId="temperatureLastReported"/>
+        </channels>
+    </channel-group-type>
+
+    <!-- The channels themselves -->
+    <channel-type id="location">
+        <item-type>Location</item-type>
+        <label>Location</label>
+        <description>Location of the device</description>
+        <state readOnly="true" pattern="%2$s°N,%3$s°W, %1$s m"/>
+    </channel-type>
+
+    <channel-type id="name">
+        <item-type>String</item-type>
+        <label>Name</label>
+        <description>Name of the device</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="exposure">
+        <item-type>String</item-type>
+        <label>Exposure</label>
+        <description>Exposure of the device (indoor/outdoor)</description>
+        <state readOnly="true">
+            <options>
+              <option value="indoor">Indoor</option>
+              <option value="outdoor">Outdoor</option>
+            </options>
+        </state>
+    </channel-type>
+    
+    <channel-type id="apiUrl" advanced="true">
+        <item-type>String</item-type>
+        <label>API Url</label>
+        <description>URL to obtain data from</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="imageUrl" advanced="true">
+        <item-type>String</item-type>
+        <label>Image Url</label>
+        <description>URL to the picture of the box on the map</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="mapUrl" advanced="true">
+        <item-type>String</item-type>
+        <label>Map Url</label>
+        <description>URL to the box on the map</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <!--
```

Comment:
Comments for the developer. Especially at the measurements, I looked at the response what units the data is given in so I don't forget that information.